.github/workflows/test_and_deploy.yml

```diff
@@ -1,19 +1,18 @@
 # This workflows will upload a Python Package using Twine when a release is created
 # For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
-
 name: tests
 
 on:
   push:
     branches:
       - main
-      - main
+      - npe2
     tags:
       - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
   pull_request:
     branches:
       - main
-      - main
+      - npe2
```

Comment:
Same as above, I don't know why this change is in the cookiecutter, can someone confirm if it's ok to have here too?

Reply:
Same as above